docs/source/tutorials/NISAR/Nisar_access_explore.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Recommend "the name".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>the names</p>

Reply:
<p>the names</p>